mlflow/server/js/src/common/components/Spinner.tsx

```diff
@@ -51,7 +42,7 @@ const styles = {
                 transform: rotate(360deg);
             }
           `} 3s linear infinite`,
-      animationDelay: immediate ? 0 : '0.5s',
+      animationDelay: immediate ? '0' : '0.5s',
```

Comment:
i think if this is a string it should be `0s`

```suggestion
      animationDelay: immediate ? '0s' : '0.5s',
```

Reply:
fixed